x-pack/plugins/index_management/public/application/sections/home/template_list/template_list.tsx

```diff
@@ -218,77 +220,99 @@ export const TemplateList: React.FunctionComponent<RouteComponentProps<MatchPara
     </>
   );
 
-  const renderContent = () => {
-    if (isLoading) {
-      return (
-        <SectionLoading>
+  // Track component loaded
```

Comment:
nit: wouldn't it be "templates" loaded?

Reply:
Hmm this pre-existing so I can't be certain, but it looks like this code will execute when the component mounts, regardless of whether the templates are loading, loaded, or failed to load. So I have to assume that by "loaded" the original author meant "mounted". Fixed.

Reply:
No I meant that the tracking. `UIM_TEMPLATE_LIST_LOAD` is for "index template loaded" not "component templates loaded" 😊